monitoring/accessing-third-party-monitoring-apis.adoc

```diff
@@ -21,6 +21,10 @@ To avoid these issues, follow these recommendations:
 
 // Accessing service APIs for third-party monitoring components
 include::modules/monitoring-about-accessing-monitoring-web-service-apis.adoc[leveloffset=+1]
+
+.Additional resources
+xref:../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[Reviewing monitoring dashboards]
```

Comment:
```suggestion
* xref:../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[Reviewing monitoring dashboards]
```

Reply:
I think the links need to be in an unordered list. I'm aware this isn't a list, but I think the bullet is needed still. Correct me if I'm wrong.

Reply:
@skrthomas - [You're right](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections)! Good catch.